test/integration/scripts/get-api-id.sh

```diff
@@ -1,6 +1,13 @@
 #! /bin/bash
 
+FILE=./test/integration/identifiers/api-id
+if [ -f "$FILE" ]; then
+    cat $FILE
+    exit 0
+fi
+
 api=$( auth0 apis create --name integration-test-api-newapi --identifier http://integration-test-api-newapi --scopes read:todos --json --no-input )
 
 mkdir -p ./test/integration/identifiers
-echo "$api" | jq -r '.["id"]' > ./test/integration/identifiers/api-id
+echo "$api" | jq -r '.["id"]' > $FILE
+cat $FILE
```

Comment:
This change to the `get-api-id` script is the same for all other scripts and represents the core of the changes in this PR.